pkg/crc/cluster/clusteroperator.go

```diff
@@ -83,7 +83,7 @@ func getStatus(ocConfig oc.Config, selector []string) (*Status, error) {
 		}
 	}
 	if !found {
-		return nil, errors.New("no cluster operator found")
+		return cs, errors.New("no cluster operator found")
```

Comment:
This is similar to what other error path are doing in this method, so ok. However, if a caller is not checking for err before using the returned `Status`, then it's a bug in the caller. Being aware of this misbehaviour through a crash rather than having the program keep running while doing weird things is in a way a good thing :)

`return &Status{}` be better in all the error branches, as `cs` is initialized with `Available: true` which may be unwanted in error cases.

Reply:
with the refactorings I did, at the end, we can get rid of this. I added a commit on top to return nil status in case of an error.

The issue was here: https://github.com/code-ready/crc/blob/master/pkg/crc/machine/machine.go#L558. Now we properly handle this case.